theseus/geometry/functional/lie_group.py

```diff
@@ -46,7 +46,8 @@ def op(
         ) -> torch.Tensor:
             if jacobians is not None:
                 check_jacobians_list(jacobians)
-                jacobians.append(jop_autograd_fn(input)[0])
+                jacobians_op = jop_autograd_fn(input)[0]
```

Comment:
Why this change? Seems unnecessary. Maybe this was used during debugging?

Reply:
> Why this change? Seems unnecessary. Maybe this was used during debugging?

There was a bug in previous code since `jop_autograd_fn(input)[0]` is `List[torch.Tensor]`.